### PR TITLE
remove dash robot discount info from email

### PR DESCRIPTION
**Pull request by clareconstantine**
Remove Wonder Workshop Dash robot discount code from exit survey email.

cc: @ryansloan

### CODE REVIEW

File: dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml [hunk 1]
```diff
@@ -44,29 +44,6 @@
     = code_org_link + "'s"
     CS Fundamentals workshop.
 
-  - if @dash_code
```
Comment (asherkach) [inline dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml]: Can we remove setting `@dash_code = CDO.pd_workshop_exit_survey_dash_code` from `dashboard/app/mailers/pd/workshop_mailer.rb` as part of this change? Assuming so, can we clean up the `CDO.pd_workshop_exit_survey_dash_code` secret?

**Comment by clareconstantine** [comment]
I'm merging this now because it's been sitting around for a few days, and I'll follow up with a PR to remove the underlying code and `pd_workshop_exit_survey_dash_code`.

**Comment by clareconstantine** [comment]
[Remove Dash robot text from email](https://trello.com/c/pZdLDZvi/396-remove-dash-robot-text-from-email)

**Comment by clareconstantine** [comment]
[Remove Dash robot text from email](https://trello.com/c/pZdLDZvi/396-remove-dash-robot-text-from-email)

**Pull request merged**